### DISCUSSION TITLE
ComboBox doesn't set default SelectedIndex

### DISCUSSION THREAD

**Discussion by vo1s** [AvaloniaUI/Avalonia#13988] [Q&A] [answered]
I have a Combox showing after clicking a button. Before that it's not visible
```
<ComboBox 
                            SelectedIndex="{Binding MainCameraCollectionSettingsIndex}"
                            SelectedItem="{Binding CameraSelectedItem, Mode=TwoWay}"
                            ItemsSource="{Binding MainCameraCollection, Mode=TwoWay}"
                        />
```
When I show it I set `MainCameraCollectionSettingsIndex` to 0
```
public void ViewCameraCommandOn()
    {
        ...
        MainCameraCollectionSettingsIndex = 0;
        ...
```
The problem is that **it is always unselected when loaded**
![image](https://github.com/AvaloniaUI/Avalonia/assets/46219207/dbcd6bd6-fbc1-4f28-bef6-2fdec1d8bb75)
If I initialize MainCameraCollectionSettingsIndex = 0 at the start of application it doesnt help neither.

**Comment by timunie** [comment, marked as answer]
Either bind to SelectedItem or SelectedIndex
